Cargo.toml

```diff
@@ -47,8 +47,8 @@ rfd = "0.14.1"
 
 ## Configuration for `cargo packager`
 [package.metadata.packager]
-product_name = "Moxin"
-identifier = "com.moxin-org.moxin"
+product_name = "Moly"
+identifier = "com.moxin-org.moly"
```

Comment:
why retain the `moxin-org` moniker? might as well change everything. You can rename github orgs too

Reply:
About the GitHub org rename, IIRC @guofoo mentioned that Moxin remains the name for the umbrella around these projects (moly, mofa, etc.), so things like GitHub org and the website remain the same.

Reply:
yeah I made this comment before I knew that. Oops lol. Ignore